otelcolbuilder/.otelcol-builder.yaml

```diff
@@ -197,23 +197,23 @@ replaces:
   # github.com/open-telemetry/opentelemetry-log-collection/pipeline.Config.Build({{0x9aeb030, 0xc00074a090}, {0xc00071c510, 0x3, 0x3}}, 0x2?)
   #         github.com/open-telemetry/opentelemetry-log-collection@v0.26.0/pipeline/config.go:83 +0x4c5 fp=0xc0005c4c98 sp=0xc0005c4b28 pc=0x7212ee5
   # github.com/open-telemetry/opentelemetry-collector-contrib/internal/stanza.createLogsReceiver.func1({0x0?, 0xffffffffff?}, {{0xc0000f5d40, {0x9a92fc0, 0xc000b250a0}, {0x9a9e020, 0xca100b8}, 0x0}, {{0x8fe90d1, 0xc}, ...}}, ...)
-  #         github.com/open-telemetry/opentelemetry-collector-contrib/internal/stanza@v0.47.0/factory.go:81 +0x57d fp=0xc0005c4fb8 sp=0xc0005c4c98 pc=0x724a89d
+  #         github.com/open-telemetry/opentelemetry-collector-contrib/internal/stanza@v0.48.0/factory.go:81 +0x57d fp=0xc0005c4fb8 sp=0xc0005c4c98 pc=0x724a89d
```

Comment:
I'd leave those comments as-is

Reply:
Reverted to the original version of the comments, with `0.46.0`.